api/environments/identities/models.py

```diff
@@ -136,6 +151,8 @@ def generate_traits(self, trait_data_items, persist=False):
 
         if persist:
             Trait.objects.bulk_create(trait_models)
+            # Update the Identity document in dynamodb
+            self.send_to_dynamodb()
```

Comment:
I assume this is not a permanent solution?

Reply:
So this idea is to store identity and traits only in dynamo, right? we will create another lambda to do that

Reply:
No, we will need to do that using hooks on the django models eventually (and overriding the `bulk_create` method...). But I'd like to do it asyncronously as soon as possible so it needs to be totally decoupled from the API request.